### PR TITLE
Integration of Racket

### PR DESCRIPTION
**Pull request by pmatos**
This is a request to integrate Racket with oss-fuzz.

Racket is a general-purpose programming language as well as the world’s first ecosystem for language-oriented programming. It supported by a diverse community and ran by the PLT team (https://www.racket-lang.org/team.html).

I am a regular contributor to Racket and SamTH (cc'ed) is part of the core Racket team.

### CODE REVIEW

**Comment by pmatos** [comment]
@googlebot I signed it!

File: projects/racket/project.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+homepage: "https://racket-lang.org"
+primary_contact: "pocmatos@gmail.com"
+auto_ccs:
+  - samth@racket-lang.org
+sanitizers:
+ - undefined
+ - memory:
+    experimental: True
+ - address:
+    experimental: True
```
Comment (inferno-chromium) [inline projects/racket/project.yaml]: ASan is a core sanitizer, we don't recommend using an experimental flag here.
Also, we no longer recommend experimental for anything (ideally should just run the fuzzer locally with that config for a few secs), can you remove it from the two sanitizers.

**Comment by inferno-chromium** [comment]
@pmatos - we will merge after your fix that comment above.

File: projects/racket/project.yaml (hunk 1 above)
Comment (pmatos) [reply projects/racket/project.yaml]: Done - thanks for the comments.

**Comment by pmatos** [comment]
@inferno-chromium I have patched it up to remove experimental flags.

**Pull request merged**